test/models/patient_test.rb

```diff
@@ -78,6 +81,31 @@ class PatientTest < ActiveSupport::TestCase
     end
   end
 
+  describe 'pledge_summary' do
+    it "should return sent and remaining pledge for the next 7 days" do
+      # Doesn't error when there are no pregnancies
```

Comment:
How would one `destroy all existing patients and their embedded pregnancies` within that test block?